tools/accuracy_checker/configs/brain-tumor-segmentation-0002.yml

```diff
@@ -0,0 +1,42 @@
+models:
+  - name: brain-tumor-segmentation-0002
+
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   public/brain-tumor-segmentation-0002/FP32/brain-tumor-segmentation-0002.xml
+        weights: public/brain-tumor-segmentation-0002/FP32/brain-tumor-segmentation-0002.bin
+        adapter:
+            type: brain_tumor_segmentation
+
+      - framework: dlsdk
+        tags:
+          - FP16
+        model:   public/brain-tumor-segmentation-0002/FP16/brain-tumor-segmentation-0002.xml
+        weights: public/brain-tumor-segmentation-0002/FP16/brain-tumor-segmentation-0002.bin
+        adapter:
+            type: brain_tumor_segmentation
+
+    datasets:
+      - name: BraTS_2017
+        preprocessing:
+          - type: swap_modalities
+            modality_order: [1,2,3,0]
+          - type: crop_brats
+          - type: normalize_brats
+            masked: True
```

Comment:
masked should be 'ignore', I guess, if the logic hasn't changed since last 2 weeks.

Reply:
Thanks, fixed